enterprise_access/apps/api_client/lms_client.py

```diff
@@ -137,7 +137,7 @@ def enterprise_contains_learner(self, enterprise_customer_uuid, learner_id):
             results = json_response.get('results')
             results = results and results[0]
             if results and results['enterprise_customer']['uuid'] == ec_uuid and results['user']['id'] == learner_id:
-                result = True
+                result = results
```

Comment:
there's some weird assumptions here about multiple learner records which doesn't make a lot of sense to me since we're filtering by user ID _**and**_  customer... so when could there ever be more than one? Unless it's an empty array sort of situation? But I would have to look closer

Reply:
@alex-sheehan-edx There could be a situation where the person is a member of multiple groups in the same enterprise